.github/workflows/proof_ci_resources/config.yaml

```diff
@@ -1,7 +1,8 @@
-cadical-tag: latest
-cbmc-version: "5.95.1"
-cbmc-viewer-version: latest
-kissat-tag: latest
-litani-version: latest
+# Use exact versions (instead of "latest") so we're not broken by surprise upgrades.
+cadical-tag: "rel-2.0.0" # tag of latest release: https://github.com/arminbiere/cadical/releases
+cbmc-version: "6.0.0" # semver of latest release: https://github.com/diffblue/cbmc/releases
+cbmc-viewer-version: "3.8" # semver of latest release: https://github.com/model-checking/cbmc-viewer/releases
+kissat-tag: "rel-3.1.1" # tag of latest release: https://github.com/arminbiere/kissat/releases
+litani-version: "1.29.0" # semver of latest release: https://github.com/awslabs/aws-build-accumulator/releases
```

Comment:
Do you have any automation in place to be notified about available upgrades?

Reply:
I signed the team account to be notified of new CBMC releases